### PR TITLE
Removal of IPluginConfigurationPage

### PR DESCRIPTION
**Pull request by BaronGreenback**
Whilst debugging noticed that this code never returned any values - and couldn't find any instance of IPluginConfigurationPage on github.

Emby describes it as

"IPluginConfigurationPage
If your plugin requires an html configuration page within the Dashboard, implement this interface. A full guide for this is coming soon."

We use IHasWebPages for this functionality.

### CODE REVIEW

**Comment by BaronGreenback** [comment]
https://github.com/jellyfin/jellyfin-web/blob/80ea564fa199533118ef93f54b873be204bf4c8f/src/scripts/libraryMenu.js#L527

**Comment by BaronGreenback** [comment]
jf-web#2327

File: MediaBrowser.Controller/Plugins/IPluginConfigurationPage.cs [hunk 1]
```diff
@@ -3,36 +3,6 @@
...
-        /// <summary>
-        /// Gets the HTML stream.
-        /// </summary>
-        /// <returns>Stream.</returns>
-        Stream GetHtmlStream();
-    }
-
     /// <summary>
     /// Enum ConfigurationPageType.
     /// </summary>
```
Comment (barronpm) [inline MediaBrowser.Controller/Plugins/IPluginConfigurationPage.cs:8]: File should be renamed to reflect the type in it.

Comment (BaronGreenback) [reply MediaBrowser.Controller/Plugins/IPluginConfigurationPage.cs]: Removed the file. Did a bit more checking. 

ConfigurationPageType was only ever checked in the server API controller. Not referred to in any of the plugins on jellyfin.

Web side it seemed to be a hardcoded value, which is the default, and was optional.

**Review by Ullmie02 — APPROVED** [review]

**Review by barronpm — APPROVED** [review]

**Pull request merged**